src/uu/fmt/src/fmt.rs

```diff
@@ -140,7 +148,7 @@ fn parse_arguments(args: impl uucore::Args) -> UResult<(Vec<String>, FmtOptions)
                 ),
             ));
         }
-        fmt_opts.goal = cmp::min(fmt_opts.width * 94 / 100, fmt_opts.width - 3);
+        fmt_opts.goal = cmp::min(fmt_opts.width * 93 / 100, fmt_opts.width - 3);
```

Comment:
would it be possible to move 93 into a const  with a good variable name?
to make it more clear

Reply:
I hope it is more clear now.